autotest/cpp/test_cpl.cpp

```diff
@@ -1912,6 +1912,7 @@ namespace tut
               "RETRY_DELAY=1",
               nullptr
             };
+#if defined(__GNUC__) || (defined(_MSC_VER) && (_MSC_VER > 1800))
```

Comment:
why ? If we want to support MSVC 1800, we'd better have test passing

Reply:
Just try to figure out the crash reason. I have no Win with VS 2013 and decide to try disable some parts of test to find the crash place.

Reply:
ok, did you try running the test with Valgrind under Linux to see if anything suspicious would show up ? (in case this is not a MSVC 1800 only issue)

Reply:
No, let me try.

Reply:
Revert back